### PR TITLE
Actualize alert view

### PR DESCRIPTION
**Pull request by SAnatoliiS**

### CODE REVIEW

File: kit/popups/src/UIForeground/Columns/SecondaryColumn.tsx [hunk 1]
```diff
@@ -14,8 +22,22 @@ export function SecondaryColumn({ children, onPress, disabled, negative }: Secon
         };
     }, [disabled, negative, onPress]);
 
+    const isValid = useCheckChildren(
```
Comment (savelichalex) [inline kit/popups/src/UIForeground/Columns/SecondaryColumn.tsx]: So it's still will run even in `production`?

File: kit/popups/src/UIForeground/hooks.ts [hunk 2]
```diff
@@ -39,6 +38,52 @@ export function useMergedColumnStatus(
...
+        return true;
+    };
+}
+
+export function useCheckChildren(
+    children: React.ReactNode,
+    allowedChildrenComponents: Record<string, unknown>,
+    errorMessage: string,
+): boolean {
+    return React.useMemo(() => {
```
Comment (savelichalex) [inline kit/popups/src/UIForeground/hooks.ts]:
```suggestion
    if (!__DEV__) {
      return true;
    }
    return React.useMemo(() => {
```

File: kit/popups/src/UIForeground/Columns/SecondaryColumn.tsx (hunk 1 above)
Comment (SAnatoliiS) [reply kit/popups/src/UIForeground/Columns/SecondaryColumn.tsx]: Sorry, I did it on wrong branch 😄

**Review by sertony — APPROVED** [review]

File: kit/popups/src/UIAlert/UIAlertViewContainer.tsx [hunk 3]
```diff
@@ -142,11 +144,20 @@ const useStyles = makeStyles((actionCount: number) => ({
     },
 }));
 
-const headerStyles = StyleSheet.create({
+const useHeaderStyles = makeStyles((isNote: boolean) => ({
```
Comment (savelichalex) [inline kit/popups/src/UIAlert/UIAlertViewContainer.tsx]:
```suggestion
const useHeaderStyles = makeStyles((hasNote: boolean) => ({
```

**Review by savelichalex — APPROVED** [review]

**Pull request merged**